cluster/addons/cluster-loadbalancing/glbc/README.md

```diff
@@ -0,0 +1,103 @@
+
+```console
+$ gcloud compute project-info describe --project myproject
+```
+See [GCE documentation](https://cloud.google.com/compute/docs/resource-quotas#checking_your_quota) for how to request more.
+
+## Latency
+
+It takes ~1m to spin up a loadbalancer (this includes acquiring the public ip), and ~5-6m before the GCE api starts healthchecking backends. So as far as latency goes, here's what to expect:
+
+Assume one creates the following simple Ingress:
+```yaml
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: test-ingress
+spec:
+  backend:
+    # This will just loopback to the default backend of GLBC
+    serviceName: glbc-default-backend
+    servicePort: 80
+```
+
+* time, t=0
+```console
+$ kubectl get ing
+NAME           RULE      BACKEND                   ADDRESS
+test-ingress   -         glbc-default-backend:80
+$ kubectl describe ing
+No events.
+```
+
+* time, t=1m
+```console
+$ kubectl get ing
+test-ingress   -         glbc-default-backend:80   130.211.5.27
+
+$ kubectl describe ing
+target-proxy:		k8s-tp-default-test-ingress
+url-map:		    k8s-um-default-test-ingress
+backends:		    {"k8s-be-32342":"UNKNOWN"}
+forwarding-rule:	k8s-fw-default-test-ingress
+Events:
+  FirstSeen	LastSeen	Count	From				SubobjectPath	Reason	Message
+  ─────────	────────	─────	────				─────────────	──────	───────
+  46s		46s		1	{loadbalancer-controller }			    Success	Created loadbalancer 130.211.5.27
+```
+
+* time, t=5m
+```console
+$ kubectl describe ing
+target-proxy:		k8s-tp-default-test-ingress
+url-map:		    k8s-um-default-test-ingress
+backends:		    {"k8s-be-32342":"HEALTHY"}
+forwarding-rule:	k8s-fw-default-test-ingress
+Events:
+  FirstSeen	LastSeen	Count	From				SubobjectPath	Reason	Message
+  ─────────	────────	─────	────				─────────────	──────	───────
+  46s		46s		1	{loadbalancer-controller }			    Success	Created loadbalancer 130.211.5.27
+
+```
+
+## Disabling GLBC
+
+Since GLBC runs as a cluster addon, you cannot simply delete the RC. The easiest way to disable it is to do as follows:
+
+* IFF you wat to tear down existing L7 loadbalancers, hit the /delete-all-and-quit endpoint on the pod:
+
+```console
+$ kubectl get pods --namespace=kube-system
+NAME                                               READY     STATUS    RESTARTS   AGE
+glbc-7bb21                                         1/1       Running   0          1h
+$ kubectl exec glbc-7bb21 -c glbc curl http://localhost:8081/delete-all-and-quit --namespace=kube-system
+$ kubectl logs glbc-7b221 -c glbc --follow
+...
+I1007 00:30:00.322528       1 main.go:160] Handled quit, awaiting pod deletion.
+```
+
+* Nullify the RC
```

Comment:
maybe add a parenthetical (but don't delete it or the addon controller will "fix" it for you)